docs/content/how-to-guides/configure-reactions/configure-azure-storagequeue-reaction/_index.md

```diff
@@ -3,8 +3,313 @@ type: "docs"
+- [Drasi CLI](/reference/command-line-interface/) 
+- [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/) (If using Azure Managed Identities)
+
+
+## Creating the Reaction
+To create a Reaction, execute the `drasi apply` command as follows:
+
+```text
+drasi apply -f my-reaction.yaml
+```
+
+The `drasi apply` command is how you create all new Drasi resources (in this case a Reaction). The `-f` flag specifies that the definition of the new Reaction is contained in the referenced YAML file `my-reaction.yaml`.
+
+## Reaction Definitions
+The YAML file passed to `drasi apply` can contain one or more Reaction definitions. Here is an example of a Storage Queue Reaction definition:
+
+```yaml
+kind: Reaction
+apiVersion: v1
+name: my-reaction
+spec:
+  kind: StorageQueue
+  queries:
+    query1:
+    query2:  
+  properties:    
+    queueName: <Name of Queue>
+    format: <packed | unpacked>
+  identity:
+    kind: ConnectionString
+    connectionString: <Connection String of Azure Storage Account>  
+```
+
+In this definition: 
+- the **apiVersion** must be **v1**.
+- the **kind** property tells Drasi to create a **Reaction** resource.
+- the **spec.kind** property tells Drasi the kind of Reaction to create, in this case a **StorageQueue** Reaction. 
+- the **name** property tells Drasi the identity of the Reaction and must be unique within the scope of Reactions within the target Drasi environment. In the above example, the **name** of the Reaction is **my-reaction**.
+
+This table describes the other settings in the **spec** section of the Reaction definition:
+|Property|Description|
+|-|-|
+| queries | Specifies the set of **names** of the Continuous Queries the Reaction will subscribe to. |
+| properties.endpoint | Endpoint of the Storage Account queue service, in the form https://{account-name}.queue.core.windows.net, if not using a connection string. |
+| properties.queueName | Name of Queue. It should already exist on your storage account. |
+| properties.format | The output format for the messages that are enqueued. The can either be **packed** for the raw query output or **unpacked** for a message per result set change. The default value is **packed** |
+| identity | The service identity provider used for authentication to the Azure Storage Queue service, discussed below. |
+
+
+### Identity
+
+The reaction supports the following service identities:
+
+#### Microsoft Entra Workload ID
+
+Microsoft Entra Workload Identity enables your reaction to authenticate to Azure without the need to store sensitive credentials. It works by creating a federated identity between a [managed identity](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview) and the service account the reaction is running against.
+
+| Property | Description |
+|-|-|
+| kind | MicrosoftEntraWorkloadID |
+| clientId | The Client ID of the user managed identity.|
```

Comment:
Should we say system-assigned or user managed identity? The Entra doc that is referred to has both.

Reply:
only user managed identities are supported, that is what the federation process is about.